torch/onnx/symbolic_opset9.py

```diff
@@ -1866,6 +1866,20 @@ def hardswish(g, self):
     hardtanh_ = g.op("Div", hardtanh_, g.op('Constant', value_t=torch.tensor(6, dtype=torch.float)))
     return g.op("Mul", self, hardtanh_)
 
+
+@parse_args('v')
+def hardsigmoid(g, self):
```

Comment:
Line 1872-1880 can be replaced by 
```
return g.op('HardSigmoid', self, alpha_f=1/6)
```

Reply:
Thank you! I modified this part in the following commit 8e1fd82.
I didn't noticed there is a doc in `_graph_op` function in `onnx/utils.py`, where `alpha_f` is introduced :)